backend/src/main/java/com/project/stocks/repository/ScrappingRepository.java

```diff
@@ -0,0 +1,42 @@
+package com.project.stocks.repository;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+
+@Repository
+public class ScrappingRepository {
```

Comment:
This is not a repository. Rename it to a client and specify what the function does.